File: fizzbuzz/fizzbuzz.php
<?php

for ($i = 1; $i <= 100; $i++) {

  if (($i % 15) == 0) {
    print("fizzbuzz");
  }
    // in javascript, "else if"
  elseif (($i % 3) == 0) {
    print("fizz");
  }
    //
  elseif (($i % 5) == 0) {
    print("buzz");
  }
    //
  else {
    print("$i");
    // in javascript: print(i);
  }
  
  //print("<br>\n");
  print("\n");
}

# how to run:
# php fizzbuzz.php

File: fizzbuzz/fizzbuzz.jl
function isFizzBuzz(n)
    if n % 3 == 0 && n % 5 == 0
        return "FizzBuzz"
    elseif n % 3 == 0
        return "Fizz"
    elseif n % 5 == 0
        return "Buzz"
    else
        return n
    end
end

# run fizzbuzz for numbers 1-100:
for i in 1:100
    println(isFizzBuzz(i))
end


# how to run:
# julia fizzbuzz.jl
